PluralKit.Bot/Commands/Groups.cs

```diff
@@ -150,16 +148,29 @@ public async Task GroupDescription(Context ctx, PKGroup target)
                 var patch = new GroupPatch {Description = Partial<string>.Null()};
                 await _db.Execute(conn => _repo.UpdateGroup(conn, target.Id, patch));
                 await ctx.Reply($"{Emojis.Success} Group description cleared.");
+            }
+            else if (ctx.MatchRaw())
+            {
+                if (!target.DescriptionPrivacy.CanAccess(ctx.LookupContextFor(target.System)))
+                    throw Errors.LookupNotAllowed;
+                    
+                if (target.Description == null)
+                    if (ctx.System?.Id == target.System)
+                        await ctx.Reply($"This group does not have a description set. To set one, type `pk;group {target.Reference()} description <description>`.");
+                    else
+                        await ctx.Reply("This group does not have a description set.");
+                else await ctx.Reply($"```\n{target.Description}\n```");
             } 
             else if (!ctx.HasNext())
             {
+                if (!target.DescriptionPrivacy.CanAccess(ctx.LookupContextFor(target.System)))
```

Comment:
Can you move this change to #342?

Reply:
realized that just now as well. give me a second